src/box/lua/index.c

```diff
@@ -351,8 +385,10 @@ lbox_truncate(struct lua_State *L)
 static int
 lbox_index_stat(lua_State *L)
 {
-	if (lua_gettop(L) != 2 || !lua_isnumber(L, 1) || !lua_isnumber(L, 2))
-		return luaL_error(L, "usage index.info(space_id, index_id)");
+	if (lua_gettop(L) != 2 || !lua_isnumber(L, 1) || !lua_isnumber(L, 2)) {
+		diag_set(IllegalParams, "usage index.info(space_id, index_id)");
```

Comment:
`Usage: index.info(...)` (capital letter, colon). Here and in other places. Let's fix the messages while we are at it.